research/efficient-hrl/environments/ant.py

```diff
@@ -75,13 +75,13 @@ def _get_obs(self):
     # No cfrc observation
     if self._expose_all_qpos:
       obs = np.concatenate([
-          self.physics.data.qpos.flat[:15],  # Ensures only ant obs.
-          self.physics.data.qvel.flat[:14],
+          self.data.qpos.flat[:15],  # Ensures only ant obs.
```

Comment:
Is there a way to get this to work regardless of mujoco 2.0 or <2.0?  Or at least make a property method that either returns self.data or self.physics.data depending on which mujoco version?

Reply:
I see, sorry for the late response on this, I got a bit busy. I modified the ant and point environments to revert back to using self.physics, and then used the mujoco version number to get the right attribute (self.model or self.sim) based on the mujoco version. Let me know if this is more or less what you meant by this.